base_layer/core/src/common/mod.rs

```diff
@@ -28,7 +28,7 @@ use crate::consensus::DomainSeparatedConsensusHasher;
 
 pub mod borsh;
 pub mod byte_counter;
-pub mod limited_reader;
+// pub mod limited_reader;
```

Comment:
Would suggest either removing altogether or adding a quick note for why it's kept under the comment.